cfgov/jinja2/v1/_includes/molecules/content-image.html

```diff
@@ -0,0 +1,87 @@
+{# ==========================================================================
+
+    Image Inset
+
+    ==========================================================================
+
+    Description:
+
+    Creates image inset markup when given:
+
+    value:                Object defined from a StreamField block.
+
+    value.image_width:    String representing the image width.
+                          Possible values: (170, 270, 470).
+                          Defaults to 270.
+
+    value.image_position: String indicating how to float the image.
+                          Possible values: (left, right).
+                          Defaults to right.
+
+    value.text:           String of caption text to be displayed below image.
```

Comment:
Should this description include `(Optional)`?

Reply:
Yeah, I'll add that.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4177/commits/32f5fcda7d16039558b597d75496a5d12e7f3736